test/test_proxy_tensor.py

```diff
@@ -1128,7 +1130,7 @@ def f(x, mask, params, buffers):
                 for s in p.shape:
                     guard_int(s)
             x = x[mask]
-            constrain_range(x.shape[0], min=1)
+            constrain_as_value(x.shape[0], min=1)
```

Comment:
Does `constrain_as_value` show up in the graph too? Hard to tell here.

Reply:
It will show up as aten.sym_constrain_range